9.2-2.3/alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM postgres:9.2-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION 2.3.1
+ENV POSTGIS_SHA256 0f1e0fe08f55b6db791d8600bb343e4f77590725acbbec99094a30b8b7439cd8
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
```

Comment:
In addition, the layer where you run `apk upgrade` and build PostGIS is 105MB, bigger than any other layer in the image. I'd be curious to know how much of this is coming from additional packages that are needed for PostGIS and how much is coming from packages that are being upgraded.